test/migration.go

```diff
@@ -17,8 +17,8 @@ type MigrationOptions struct {
 }
 
 // RunMigrationContainer creates the migration container and connects to the
-// mysql database to run the migration scripts.
-func RunMigrationContainer(mysql *MySQLContainer, options MigrationOptions) error {
+// sql database container to run the migration scripts.
+func RunMigrationContainer(sql *SQLContainer, options MigrationOptions) error {
```

Comment:
`sql` sounds like a sql string, can you rename it as dbContainer or something else?

Reply:
OK